recipes/motor/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "motor" %}
+{% set version = "1.2.1" %}
+{% set sha256 = "ed4aff66c625efe01398e8416fd1f35f00be77cc5afb242172ee3d03a0307c04" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: "{{ sha256 }}"
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
+    - pip
+    - aiohttp
+    - tornado
+    - pymongo >=3.4,<4
+    - python >=2.7,<3.7
```

Comment:
Packages can be declared `noarch`, which makes them go into the `noarch` subdir (=platform) of the channel. Since pure Python packages are the same for any architecture, that saves space storing the package, time building it, and just generally gets rid of the redundancy.

Packages that are not noarch, will be built once for each platform: linux-64, win-64, osx-64 (IIRC win-32 was dropped by CF a while ago). You can skip some of those if the package doesn't support the platform.

I'll leave explaining about Perl/Lua/R/Python to @scopatz because I don't think I've fully grasped the new CB3 behavior there.

Reply:
The changes have been made, the constraints are added to the skip selector